tests/TestRunner/TestRunner.py

```diff
@@ -305,7 +304,7 @@ def run_simulation_test(basedir, options):
         return_codes[command] = proc.returncode
         if proc.returncode != 0:
             break
-    outfile = os.path.join(test_dir, 'traces.{}'.format(options.log_format))
```

Comment:
@mpilman is there a reason we have test_dir instead of basedir here? I maybe missed some discussion

Reply:
Yes see #1076 and #1058 - I realize this is inconvenient for loading the data into snowflake, but it shouldn't be too hard to find the test directory after the run. So please revert this as this will otherwise break other the workflow of people.

Reply:
Done